src/shared/analytics/analyticsId.client.ts

```diff
@@ -0,0 +1,8 @@
+import { BrowserStorage } from 'src/background/webapis/storage';
+import { walletPort } from 'src/ui/shared/channels';
+import { analyticsIdKey } from './analyticsId';
+
+export async function setAnalyticsIdIfNeeded(id: string) {
```

Comment:
what does `ifNeeded` imply?

Reply:
Thank you! This was left from the initial implementation